LmDataset.py

```diff
@@ -1009,6 +1009,7 @@ def _data_str_to_numpy(self, vocab, s):
     :rtype: numpy.ndarray
     """
     words = s.split()
+
```

Comment:
Why?

Reply:
You told me to do it! To put splitting into words inside the vocabulary class!

Reply:
I don't understand. The only change here is adding a newline. I was asking, why is adding a newline needed here. Better remove this newline.